packages/flutter/pubspec.yaml

```diff
@@ -5,7 +5,9 @@ description: A framework for writing Flutter applications
 homepage: http://flutter.io
 
 dependencies:
+  async: "^1.10.0"
   collection: '>=1.9.1 <2.0.0'
+  http_parser: ">=0.0.1 <4.0.0"
```

Comment:
Can you add a comment that we can remove these once we switch to using `package:http`?